spec/services/data_requests/write_cloudwatch_logs_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'csv'
```

Comment:
I feel like we shouldn't add this in the specs, the services should require the files?

Reply:
I think the only reason why I was thinking here was because it uses CSV directly in the test but you're right. It's included in the test subject which is probably sufficient.